big_tests/tests/domain_removal_SUITE.erl

```diff
@@ -197,6 +202,33 @@ private_removal(Config) ->
         ?assert_equal_extra(<<>>, Val2, #{stanza => Res2})
       end).
 
+bobs_default_groups() -> [<<"friends">>].
+
+bobs_default_name() -> <<"Bobby">>.
+
+roster_removal(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
+        %% add contact
+        Stanza = escalus_stanza:roster_add_contact(Bob, bobs_default_groups(),
```

Comment:
Why here is used `bobs_default_groups/0` and `bobs_default_name/0`? Maybe make sense change to:
```erlang
Stanza = escalus_stanza:roster_add_contact(Bob, [<<"friends">>], <<"Bobby">>),
```